.eslintrc.json

```diff
@@ -17,9 +17,12 @@
     "EprotectIframeClient": true,
     "ipaddr": true,
     "JSURL": true,
+    "JSZip": true,
+    "JSZipUtils": true,
     "moment": true,
     "punycode": true,
     "RandExp": true,
+    "saveAs": true,
```

Comment:
Missing from `package.json`

Reply:
saveAs is included in jszip vendor repository

Reply:
Would it make sense to `import` where it is used ?